cmake/ctest/drivers/atdm/ats2/local-driver.sh

```diff
@@ -21,12 +21,12 @@ atdm_run_script_on_compute_node \
   ${LSF_CTEST_TIMEOUT}
 
 if [ "${Trilinos_CTEST_RUN_CUDA_AWARE_MPI}" == "1" ]; then
+  # Wait a little while before attempting another bsub allocation
+  # to try working around "'Error: Remote JSM server is not responding'"
+  sleep 5
   export TPETRA_ASSUME_CUDA_AWARE_MPI=1
   export CTEST_BUILD_NAME=${ATDM_CONFIG_BUILD_NAME}_cuda-aware-mpi
-  export CTEST_DO_UPDATES=FALSE
```

Comment:
@e10harvey, I think you want to avoid the updates as that will be confusing for CDash.  It is okay to do the configure and build though.

Reply:
This export is redundant due to https://github.com/trilinos/Trilinos/pull/6724/files#diff-a95886e3c981a2798804e59c33701ac4L18.

Reply:
Right, that is correct.  That is calling the "test" driver script.  Thanks for pointing that out.